.shellcheckrc

```diff
@@ -0,0 +1,4 @@
+# Allow which usage
```

Comment:
I would leave a link to the guide here. To have a quick reference and find rules, for example, `SC2230` :)

Reply:
Good, added